velox/exec/OutputBuffer.cpp

```diff
@@ -637,7 +637,7 @@ double OutputBuffer::getUtilization() const {
 }
 
 bool OutputBuffer::isOverutilized() const {
-  return (totalSize_ > maxSize_) && !atEnd_;
+  return (totalSize_ > (0.5 * maxSize_)) && !atEnd_;
```

Comment:
Good catch. Does Java set the over-utilized flag based on the same parameter? Thanks!

Reply:
Yes. See com/facebook/presto/execution/buffer/ArbitraryOutputBuffer.java

```
    @Override
    public boolean isOverutilized()
    {
        return (memoryManager.getUtilization() >= 0.5) || !state.get().canAddPages();
    }

    public double getUtilization()
    {
        return bufferedBytes.get() / (double) maxBufferedBytes;
    }

    private boolean isBufferFull()
    {
        return bufferedBytes.get() > maxBufferedBytes && blockOnFull.get();
    }
```